# This message defines what phase of the objective that we are in
# The variable will be an int 1-5. 1 is Start, 2 is Kiss Brigham, 3 is Reverse, 4 is go home, and 5 is finished

int32 state
